text2speech.py

```diff
@@ -15,14 +16,8 @@
 
 separator = "="
 aws_properties = {}
-
```

Comment:
Would you mind changing this use environment variables but fall back to the aws.properties file if they aren't defined?